lnwallet/interface.go

```diff
@@ -80,6 +80,14 @@ type Utxo struct {
 	wire.OutPoint
 }
 
+// DestOutput contains additional information on a destination address.
+type DestOutput struct {
```

Comment:
Should we call this struc `OutputDetail`, add the `Address btcutil.Address` field to it and completely replace the `DestAddresses` below? Could call the field `OutputDetails` instead of `DestOutputs` too.

Reply:
I agree and have renamed it!